.devcontainer/devcontainer.json

```diff
@@ -4,7 +4,25 @@
     "dockerfile": "Dockerfile",
     "context": ".."
   },
-  "appPort": "8124:8123",
+  "forwardPorts": [8123, 8100, 8090, 8080],
+  "portsAttributes": {
+    "8123": {
+      "label": "Home Assistant",
+      "onAutoForward": "silent"
+    },
+    "8100": {
+      "label": "Gallery",
+      "onAutoForward": "silent"
+    },
+    "8090": {
+      "label": "Demo",
+      "onAutoForward": "silent"
+    },
+    "8080": {
+      "label": "Cast",
+      "onAutoForward": "silent"
+    }
+  },
```

Comment:
This does not belong in this PR.

Reply:
Why not? I had to change it in order to make the debugger work in the devcontainer. AppPort was causing VSCode to forward the Hass 8123 to 8124, while the debugger is configured to open on 8123.

With this change, VS Code will auto-select the best available port (8123 if available), but if it's not available, the debugger will also take the forwarded port into account and dynamically update it.

Reply:
The change doesn't have to be wrong, but a PR should change 1 thing, this should be in a separate PR so reviewing and merging is quicker and easier.

Reply:
Alright. Will do that. Thanks.